modules/View.js

```diff
@@ -680,6 +680,7 @@
           });
           behaviorOptions.view = view;
           var behaviorInstance = view.__behaviorInstances[alias] = new BehaviorClass(behaviorOptions, viewOptions);
+          behaviorInstance.alias = alias;
```

Comment:
add the alias to behaviorOptions above and set it in the constructor so its available to initialize.